### PR TITLE
Don't wait for replug of emulated devices

### PR DESCRIPTION
**Pull request by fdanis-oss**
Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [ ] Code fix
- [ ] Feature
- [ ] Documentation

### CODE REVIEW

**Comment by hughsie** [comment]
Does this make the TI dock work?

File: src/fu-device-list.c [hunk 1]
```diff
@@ -905,6 +905,13 @@ fu_device_list_wait_for_replug(FuDeviceList *self, GError **error)
 		return TRUE;
 	}
 
+	/* emulated devices don't need to wait for replug */
+	for (guint i = 0; i < devices_wfr1->len; i++) {
+		FuDevice *device_tmp = g_ptr_array_index(devices_wfr1, i);
+		if (fu_device_has_flag(device_tmp, FWUPD_DEVICE_FLAG_EMULATED))
+			fu_device_remove_flag(device_tmp, FWUPD_DEVICE_FLAG_WAIT_FOR_REPLUG);
```
Comment (hughsie) [inline src/fu-device-list.c:912]: I think get rid of the `if fu_device_has_flag(device_tmp, FWUPD_DEVICE_FLAG_EMULATED` -- just remove it unconditionally.

Comment (hughsie) [reply src/fu-device-list.c:912]: Nope, ignore me, i'm being crazy.

Comment (fdanis-oss) [reply src/fu-device-list.c:912]: if I remove the test it will break the behavior when waiting for a real device re-plug, no?

**Comment by fdanis-oss** [comment]
Yes, it allows to complete TI dock update, but there's still another issue as it doesn't get the right firmware version

**Comment by fdanis-oss** [comment]
Close as it's better handled by #5479

**Pull request closed without merging**